website/docs/docs/get-started/dbt-cloud-features.md

```diff
@@ -3,44 +3,109 @@ title: "dbt Cloud features"
+    link="/docs/get-started/develop-in-the-cloud#build-compile-and-run-projects"
+    icon="pencil-paper"/>      
+
+   <Card
+    title="Supports GitHub, GitLab, AzureDevOPs"
+    body="Seamlessly connect your git account to dbt Cloud and provide another layer of security to dbt Cloud. Import new repositories, trigger continuous integration, clone repos using HTTPS, and more!"
+    link="/docs/collaborate/git/connect-github"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="Enable Continuous Integration"
+    body="Configure dbt Cloud to run your dbt projects in a temporary schema when new commits are pushed to open pull requests. This build-on-PR functionality is a great way to catch bugs before deploying to production, and an essential tool in any analyst's belt."
+    link="/docs/deploy/cloud-ci-job"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="Security"
+    body="Manage risk with SOC-2 compliance, CI/CD deployment, RBAC, and ELT architecture."
+    link="https://www.getdbt.com/security/"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="dbt Semantic Layer*"
+    body="Use the dbt Semantic Layer to define metrics alongside your dbt models and query them from any integrated analytics tool. Get the same answers everywhere, every time."
+    link="/docs/use-dbt-semantic-layer/dbt-semantic-layer"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="Metadata API*"
+    body="Use the Metadata API to enhance your workflow and run ad-hoc queries, browse schema, or query the dbt Semantic Layer. dbt Cloud serves a GraphQL API, which supports arbitrary queries."
+    link="/docs/dbt-cloud-apis/metadata-api"
+    icon="pencil-paper"/> 
+
+
+<Card
+    title="Model timing dashboard*"
+    body="The Model timing dashboard visualizes and lets you explore your run and surface model bottlenecks. The dashboard shows model info, order, and run time for each job completed. The display only appears for successfully completed jobs, and the top 1% of model times are highlighted. Access the dashboard on the Run Overview page in dbt Cloud."
+    link="/docs/dbt-versions/release-notes/January-2022/model-timing-more"
+    icon="pencil-paper"/> 
+</div> <br />
+
+ ***These features are available on [select plans](https://www.getdbt.com/pricing/).**
+
+## Related questions
+
+<details>
+  <summary>How can I contribute to dbt Cloud?</summary>
+  <div>
+    <div>Anyone can contribute to the dbt project. And whether it's a dbt package, a plugin, dbt-core, or this documentation site, contributing to the open source code that supports the dbt ecosystem is a great way to level yourself up as a developer, and give back to the community. See <a href="https://docs.getdbt.com/community/resources/oss-expectations">Contributing</a> for details on what to expect when contributing to the dbt open source software (OSS). </div>
+  </div>
```

Comment:
sugg: does it make sense to remove this and add a "Become a contributor" link to  https://docs.getdbt.com/community/contribute in **Related docs** section?